packages/store/src/StoreCore.sol

```diff
@@ -372,6 +362,34 @@ library StoreCore {
     }
   }
 
+  /************************************************************************
+   *
+   *    EPHEMERAL SET DATA
+   *
+   ************************************************************************/
+
+  /**
+   * Emit the ephemeral event without modifying storage for the full data of the given tableId and key tuple
+   */
+  function setEphemeralRecord(bytes32 tableId, bytes32[] memory key, bytes memory data) internal {
```

Comment:
Do we need to add `setEphemeralField` too?

Reply:
I'd favor only record, by analogy to events it'd be like emitting an event with partial arguments, which probably means you just want more events, or an argument in the event itself to specify which arguments to look at